wherehows-web/tests/acceptance/login-test.js

```diff
@@ -25,16 +38,73 @@ test('should render login form', function(assert) {
   });
 });
 
-test('should display error message with empty credentials', async function(assert) {
-  assert.expect(2);
+// test('should display error message with empty credentials', async function(assert) {
+//   assert.expect(2);
+//   await fillIn(loginUserInput, testUser);
+//   await click('button[type=submit]');
+//
+//   assert.ok(find('#login-error').text().length, 'error message element is rendered');
+//
+//   assert.equal(
+//     find('#login-error')
+//       .text()
+//       .trim(),
+//     invalidCredentials
+//   );
+// });
 
-  await click('button[type=submit]');
+test('Login with an valid password', function(assert) {
+  visit('/#/login');
+  fillIn(loginUserInput, testUser);
+  fillIn(loginPasswordInput, testPassword);
+  click(loginSubmitButton);
 
-  assert.ok(find('#login-error').text().length, 'error message element is rendered');
-  assert.equal(
-    find('#login-error')
-      .text()
-      .trim(),
-    invalidCredentials
-  );
+  andThen(function() {
+    visit('/#/');
+  });
+  andThen(function() {
+    assert.equal(
+      find(`${'.feature-card__title'}:eq(0)`)
```

Comment:
Seems like this is out of the domain of a `login` test, I would move this to the tests for the browse feature

Reply:
fix